jaeger-apachehttpclient/src/test/java/com/uber/jaeger/httpclient/JaegerRequestAndResponseInterceptorIntegrationTest.java

```diff
@@ -124,7 +130,7 @@ public void testHttpClientTracing() throws Exception {
   private void verifyTracing(Span parentSpan) {
     //Assert that traces are correctly emitted by the client
     List<Span> spans = reporter.getSpans();
-    assertEquals(3, spans.size());
+    assertEquals(2, spans.size());
```

Comment:
what is the reason for this change?

Reply:
Previously, when `pop()` was getting called, it would make another call to report the span. Hence the reporter was getting an extra span, and the last 2 spans were identical. This changes proves that the core issue is fixed.